tangelo/problem_decomposition/dmet/dmet_problem_decomposition.py

```diff
@@ -182,7 +181,7 @@ def __init__(self, opt_dict):
         if not self.solvers_options:
             for solver in self.fragment_solvers:
                 if solver.lower() in {"ccsd", "fci", "mp2", "hf"}:
-                    self.solvers_options.append(default_classical_options)
+                    self.solvers_options.append(dict())
```

Comment:
Could we leave a comment here, stating the purpose you explained earlier (e.g having the same length for something) ?

If you have a suggestion for a # TODO, I think it would be great to leave as comment.